pypeit/core/telluric.py

```diff
@@ -1408,11 +1563,11 @@ def create_bal_mask(wave, bal_wv_min_max):
 
 
 
-def qso_telluric(spec1dfile, telgridfile, pca_file, z_qso, telloutfile, outfile, npca=8,
-                 pca_lower=1220.0, pca_upper=3100.0, bal_wv_min_max=None, delta_zqso=0.1,
+def qso_telluric(spec1dfile, telgridfile, teltype, pca_file, z_qso, telloutfile, outfile, npca=8,
```

Comment:
I think this should be optional and we should simply default to the 'pca'

Reply:
Made that change in all the `*_telluric` functions